src/beautifiers/eslint.coffee

```diff
@@ -9,25 +9,49 @@ module.exports = class ESLintFixer extends Beautifier
   link: "https://github.com/eslint/eslint"
 
   options: {
-    JavaScript: false
+    JavaScript:
+      errors_only: true
   }
 
   beautify: (text, language, options) ->
     return new @Promise((resolve, reject) ->
       editor = atom.workspace.getActiveTextEditor()
       filePath = editor.getPath()
-      projectPath = atom.project.relativizePath(filePath)[0]
+
+      if filePath
+        dir = path.dirname(filePath)
+        projectPath = atom.project.relativizePath(filePath)[0]
+      else
+        dir = atom.getConfigDirPath()
+        projectPath = atom.project.getPaths()[0]
 
       result = null
-      allowUnsafeNewFunction ->
-        importPath = Path.join(projectPath, 'node_modules', 'eslint')
-        try
-          CLIEngine = require(importPath).CLIEngine
+      importPath = Path.join(projectPath or dir, 'node_modules', 'eslint')
+      try
+        eslint = require(importPath)
+      catch
+        eslint = require('eslint')
```

Comment:
I don't think `require('eslint')` is sufficient to be "global". By global, you mean installed via `npm install --global eslint`?
See https://stackoverflow.com/a/15646750/2578205

Reply:
This would work if `eslint` was a dependency in Atom-Beautify's `package.json` file.

Reply:
eslint is dependency in Atom-Beautify's package.json file.

Reply:
Correct: https://github.com/Glavin001/atom-beautify/blob/master/package.json#L191

However I want to confirm the goal here. Is it to use the `eslint` installed by `npm install --global eslint` (global) or the one installed within Atom-Beautify itself? The latter is what is being done currently, however the description makes me think the former is what you actually want.
We could support all three options, however something like [`requireg`](https://www.npmjs.com/package/requireg) will be required.

Reply:
I meant global as just not local to the project. Using the eslint included with Atom-Beautify is fine with me.

Reply:
I could check for *global* eslint before the included one if that is what people want

something like:

```coffeescript
try
  eslint = require(importPath)
catch
  try
    eslint = requireg('eslint')
  catch
    eslint = require('eslint')
```

Reply:
This looks like a good idea: future proof. 👍

Reply:
looks like `requireg` checks the regular `require` first so it would always grab the included one 😕 

We could just add an eslint option to the Executables settings so people could specify there own executable if they wanted to.

I'm not sure how to add a new option under Executables